### DISCUSSION TITLE
Transitions & Fading

### DISCUSSION THREAD

**Discussion by 1212312312312** [tidal-music/discussions#69] [Ideas & Feature requests]
Listening to music on Tidal is a blast. However, I feel with Spotify on one thing, cross-fading in-between songs is a must. It's a fantastic ability to have where having that flow between your songs feels much better. As I'm rooting for Tidal to become better in any way, I would love to see this happen in near future!

**Comment by stefanmoro** [comment]
Hey @1212312312312 ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to our existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
